packages/grid-pro/src/vaadin-grid-pro-inline-editing-mixin.js

```diff
@@ -298,7 +298,11 @@ export const InlineEditingMixin = (superClass) =>
     /** @private */
     _onEditorFocusOut() {
       // Schedule stop on editor component focusout
-      this._debouncerStopEdit = Debouncer.debounce(this._debouncerStopEdit, animationFrame, this._stopEdit.bind(this));
+      this._debouncerStopEdit = Debouncer.debounce(
+        this._debouncerStopEdit,
+        timeOut.after(0),
```

Comment:
When I tested manually, adding the `mousedown` listener on the editor was enough to fix the issue, but I couldn't make the test I created pass without this change. Maybe there's an alternative approach to the test so that we can revert this change.